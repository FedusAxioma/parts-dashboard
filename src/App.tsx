import PartA from './pages/PartA';
import { PartsProvider } from './contexts/PartsContext';
import React from 'react';

const App: React.FC = () => {
	return (
		<PartsProvider>
			<PartA />
		</PartsProvider>
	);
};

export default App;
