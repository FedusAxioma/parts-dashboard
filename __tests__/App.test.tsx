import App from '../src/App';
import { createRenderer } from 'react-test-renderer/shallow';
import React from 'react';

describe('App', () => {
	const shallow = createRenderer();

	test('renders base setup', () => {
		shallow.render(<App />);

		expect(shallow.getRenderOutput()).toMatchSnapshot();
	});
});
