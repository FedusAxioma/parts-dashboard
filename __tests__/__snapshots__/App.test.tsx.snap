// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders base setup 1`] = `
<PartsProvider>
  <PartA />
</PartsProvider>
`;
